src/Features/GlobalProperties/__snapshots__/GlobalProperties.spec.js.snap

```diff
@@ -600,7 +600,7 @@ exports[`GlobalPropertiesContainer --- Snapshot Test Capturing Snapshot of Globa
                       <p
                         class="tableTextarea"
                       >
-                        •••••••••
+                        ---
```

Comment:
Is this `---` value expected?

Reply:
this would be from adjusting the data within the fixture to return null instead of a value

Reply:
got it thanks